### ISSUE TITLE
Incorrect 1st positioning of a tooltip

### ISSUE THREAD

**Issue by artemdzyuba** [atomiks/tippyjs#699]
## Bug description

When I first move my cursor over the object, the tooltip is shown in an unexpected position/ When I move the cursos away and return it, the newly shown tooltips appears correctly. I 

Please see the screenshots on my [Notion page](https://www.notion.so/artem1982/Tippsy-fab76bd3391e46159e982de4610f991b).

Unfortunately, I cannot recreate this problem wirh Codepen, as it is a part of large project with assets, which I cannot move to Codepen.

If this is important,I have tooltips firing on $(document).on(mouseover), otherwise I couldn't get them work.

## Reproduction

CodePen link: https://codepen.io/atomiks/pen/yvwQyZ

**Comment by atomiks** [comment]
> If this is important,I have tooltips firing on $(document).on(mouseover), otherwise I couldn't get them work.

I'd say that is likely the reason, avoid doing that. If you need "event delegation" behavior, then use `delegate()`

> otherwise I couldn't get them work.

I'm assuming the script is loading before the elements get created..

**Comment by artemdzyuba** [comment]
Thank you for the reply. Tried with the "delegate" method, it is not supported in jQ 3. 

> I'm assuming the script is loading before the elements get created..

Sure

I realised that this happens because of the picture. When I show plain text, the result is fine. My content code for the case is
`<img class="tooltip-image" src="img/san.png">`

I found out, that larger images make things unpredictable. I took a larger image, and it is mostly at bottom, but sometimes at top. I recorded [video](https://youtu.be/7wINAEu5VCI), at YouTube, 13 seconds.

I tried to make a tooltip instance to hide/disable or what ever not to show the first time and immediately show it for the second time, but without success. When I declare the const instance, it is already set to show.
